Modelica/Electrical/Batteries/Examples/CCCV_Cell.mo

```diff
@@ -2,10 +2,11 @@ within Modelica.Electrical.Batteries.Examples;
 model CCCV_Cell
   "Charge a cell with constant current - constant voltage characteristic"
   extends Modelica.Icons.Example;
+    parameter Modelica.Units.SI.Current Isc = 1200 "Short-circuit current of cell at OCVmax";
```

Comment:
Match indentation to the remaining lines?
```suggestion
  parameter Modelica.Units.SI.Current Isc = 1200 "Short-circuit current of cell at OCVmax";
```

Reply:
Remove blanks